salt/utils/vault.py

```diff
@@ -335,12 +335,19 @@ def make_request(
     namespace = namespace or connection.get("namespace")
     if "verify" in args:
         args["verify"] = args["verify"]
+        log.debug(
+            'TLS bundle path for Vault found in args["verify"]: %s', args["verify"]
```

Comment:
Will this ever log sensitive information?

Reply:
It's the **path** to the public key of the accepted certificate to connect to Vault, so I'd say non sensitive (the private key for the certificate would be sensitive information).

Reply:
Thanks, this looks good, can you just add a changelog and we can get this merged in